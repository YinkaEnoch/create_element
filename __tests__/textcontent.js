const c = require("../src/index.ts");

test("add text to a span element", () => {
	let el = c(
		"span",
		{ id: "author", class: "red-text" },
		"Yinka Enoch Adedokun"
	);

	expect(el).toMatchSnapshot();
});
