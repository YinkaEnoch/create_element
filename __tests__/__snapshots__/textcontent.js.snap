// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add text to a span element 1`] = `
<span
  class="red-text"
  id="author"
>
  Yinka Enoch Adedokun
</span>
`;
